### ISSUE TITLE
Best way for people to download >30 database files from PhysioNet?

### ISSUE THREAD

**Issue by tompollard** [mimic2/v3.0#64]
Users need to download over 30 compressed files from PhysioNet (each around 200MB) to reconstruct the MIMIC database. It would be helpful for us to provide advice on a quick way to download all of the files. 

One option is to use wget with the PhysioNet username - e.g. something like this:

``` bash
wget --user <USERNAME> --ask-password --no-check-certificate -r -np -A.gz  <url>
```

**Comment by SvenVanPoucke** [comment]
Hi,
What is the added value to have users to download the database? Why not providing them with an access as for the query builder but with full export facilities. Please also take a look at our approach using a hadoop cluster with radoop link to be able to use RapidMiner to analyze the database.
https://youtu.be/zToFij3SW8k
Cheers
Sven

**Comment by Franck-Dernoncourt** [comment]
I agree that having access to the [query builder](https://mimic.physionet.org/database/explorer/65-query-builder.html) with full export facilities would be enough in many cases, but giving the option to the user to download the database has some nice upsides, on the top of my head:

- using a better IDE than the online query builder (auto-completion, syntax highlighting, query formatting, more export options, more shortcuts, more options to view tables, etc.)
- querying the database directly from some other code (e.g. I often do my analyses in Python and directly retrieve data from MIMIC on the fly)
- can be used offline
- decreased load on the server
- using a better server to make query faster
- modifying the data

**Comment by SvenVanPoucke** [comment]
Dear Franck,
I fully agree!
Cheers
Sven

Op vr 31 jul. 2015 20:31 schreef Franck Dernoncourt <
notifications@github.com>:
